### PR TITLE
Add link to GitHub in footer

### PR DESCRIPTION
**Pull request by Syfaro**
It looks like this. 

The sha links to that revision on GitHub, the line of text just links to the GitHub page.

### CODE REVIEW

File: templates/common/page_end.html [hunk 1]
```diff
@@ -65,7 +65,8 @@ <h2 class="typeface-a color-a">Support</h2>
       $if active_users_string:
         ${active_users_string}.
         <br>
-      ${SHA} &middot; Copyright &copy; 2012–${arrow.now().year} Weasyl LLC
+      <a href="https://github.com/Weasyl/weasyl/commit/${FULLSHA}">${SHA}</a> &middot; Copyright &copy; 2012–${arrow.now().year} Weasyl LLC<br>
```
Comment (weykent) [inline templates/common/page_end.html]: Github allows shorthashes: https://github.com/Weasyl/weasyl/commit/327dbed

File: static/public.css [hunk 2]
```diff
@@ -4548,6 +4548,11 @@ img + #detail-art-text {
     color: #939699;
 }
 
+#footer-info a,
+#footer-info a:visited {
```
Comment (charmander) [inline static/public.css]: Is the `:visited` really necessary here? I sure hope not.

Comment (Syfaro) [inline static/public.css]: Whoops, apparently not. I added it when trying to figure out why the color wasn't changing, turned out that for some reason Safari wasn't reloading the stylesheet. Removed now.

**Comment by weykent** [comment]
:+1: I love it.

**Pull request merged**